route4me-csharp-sdk/Route4MeSDKLibrary/Route4MeManagerV5.cs

```diff
@@ -2503,9 +2940,13 @@ public OrderHistoryResponse GetOrderHistory(OrderHistoryParameters parameters, o
         /// <returns>Order history</returns>
         public Task<Tuple<OrderHistoryResponse, ResultResponse>> GetOrderHistoryAsync(OrderHistoryParameters parameters)
         {
-            return GetJsonObjectFromAPIAsync<OrderHistoryResponse>(parameters,
+            var result = GetJsonObjectFromAPIAsync<OrderHistoryResponse>(parameters,
                 R4MEInfrastructureSettingsV5.OrdersHistory,
                 HttpMethodType.Get, null, true, false);
+
+            return Task.FromResult(
```

Comment:
What is the purpose of this change? I would recommend to avoid unnecessary memory traffic here. You can just return result or revert changes.

Reply:
See the comment above for similar question.